Problems/TwoPunctures/two_punctures_robin_sphere_hp_mgpc_newton_petsc_cactus.c

```diff
@@ -476,7 +474,7 @@ problem_init
     ip_norm_data.sipg_flux_h = sipg_params->sipg_flux_h;
     ip_norm_data.penalty_prefactor = sipg_params->sipg_penalty_prefactor;
     
-    d4est_output_norms_t norms = d4est_output_norms
+    d4est_norms_norms_t norms = d4est_norms_norms
```

Comment:
d4est_norms_norms_t -> d4est_norms_t

Reply:
This struct may also be removed I think, unless the set of computed norms need to be available somewhere at runtime instead of being written to a file.

Reply:
Yeah I don't think they are really used anywhere, so it can be removed.